deploy-board/deploy_board/templates/environs/env_landing.html

```diff
@@ -209,6 +209,7 @@ <h4 class="modal-title">Resume Confirm</h4>
 <script>
 
 var capacityInfo = {{capacity_info | safe}}
+var placements = {{placements | safe}}
```

Comment:
Since the only needed value is the remaining size, it's better to calculate the remaining capacity in the Django App so the page can load faster with less contents to download and compute.